src/containers/WorkspaceRecordingView.tsx

```diff
@@ -17,20 +17,32 @@ interface Props {
   sortings: Sorting[]
   workspaceInfo: WorkspaceInfo
   plugins: Plugins
+  width: number
+  height: number
   workspaceRouteDispatch: WorkspaceRouteDispatch
 }
 
 const calculationPool = createCalculationPool({maxSimultaneous: 6})
 
-const WorkspaceRecordingView: FunctionComponent<Props> = ({ recording, sortings, workspaceInfo, plugins, workspaceRouteDispatch }) => {
+const WorkspaceRecordingView: FunctionComponent<Props> = ({ recording, sortings, workspaceInfo, plugins, workspaceRouteDispatch, width, height }) => {
   const { readOnly } = workspaceInfo;
   const recordingInfo = useRecordingInfo(recording.recordingObject)
-  if (!recordingInfo) return <div>Loading recording info</div>
 
   const handleImportSortings = () => {
     console.info('not yet implemented')
   }
 
+  const initialRecordingSelection: RecordingSelection = {}
+  const [selection, selectionDispatch] = useReducer(recordingSelectionReducer, initialRecordingSelection)
+
+  useEffect(() => {
+    if ((!selection.timeRange) && (recordingInfo)) {
+      selectionDispatch({type: 'SetTimeRange', timeRange: {min: 0, max: Math.min(recordingInfo.num_frames, recordingInfo.sampling_frequency / 10)}})
```

Comment:
Pretty sure we have similar 'default time range' code elsewhere; should probably be added to a utility library as a function.

Reply:
creating new issue for this.